### DISCUSSION TITLE
Connect multiple functions to a single chat cell?

### DISCUSSION THREAD

**Discussion by reviewsbyryan** [Ironclad/rivet#157] [Q&A] [answered]
Hello, is it possible to connect multiple functions to a chat cell? When configuring the chat cell, enabling functions causes the 'functions' point to become visible, but you cannot actually connect multiple functions to it, only one.

**Comment by reviewsbyryan** [comment, marked as answer]
Solved: Connect multiple functions to an array and pass the array as the list of functions.

**Reply by adp60** [reply]
Hi, quick question here, what are functions in rivet and what is their use? Total noob here.. trying to get more out of this wonderful app.

**Reply by abrenneke** [reply]
It's basically this https://platform.openai.com/docs/guides/gpt/function-calling
